internal/install/recipe_installer.go

```diff
@@ -259,18 +243,20 @@ func (i *RecipeInstaller) installRecipesWithPrompts(m *types.DiscoveryManifest,
 		}
 
 		log.Debugf("Executing and validating with progress for recipe name %s...", r.Name)
+
+		// TODO Handler the error here.  The tests indicate that this method should
```

Comment:
Do we really need to do anything else here?  We are handling the error currently, by swallowing it & logging.

Reply:
Oh yeah, this has been resolved.